.cirrus.yml

```diff
@@ -48,7 +48,7 @@ task:
     fingerprint_script: echo $CIRRUS_BUILD_ID
   build_detox_script: npm run build-detox-android
   cleanup_before_cache_script:
-    - rm -rf ~/.gradle/caches/6.2/
+    - rm -rf ~/.gradle/caches/6.9.2/
```

Comment:
Based on what I see in this folder on my machine, this should be `~/.gradle/caches/6.9` - e.g. no patch number.

Reply:
if you rerun the android project (e.g. `npm run android`), you should see a cache folder for `6.9.2` too. I think the `6.9` was created from previous runs of the android commands when we had the version set to 6.9 (i.e. before this PR's change)

Reply:
oh right! yeah, makes sense. Let's leave as-is then.

Reply:
when you added this to Cirrus, we were on `6.2` so I assumed that this step should be removing the cache of the currently used gradle version. lmk if that's incorrect though